Mathlib/Analysis/Fourier/FourierTransformDeriv.lean

```diff
@@ -767,7 +767,13 @@ lemma hasDerivAt_fourierIntegral
     apply ContinuousLinearMap.ext_ring
     rw [ContinuousLinearMap.smulRight_apply, ContinuousLinearMap.flip_apply,
       ContinuousLinearMap.mul_apply', one_mul, ContinuousLinearMap.map_smul]
-    exact congr_arg (fun x ↦ v • x) (one_smul ℝ (f v)).symm
+    expose_names
```

Comment:
the `with v` five lines above doesn't work, this is why this is necessary... somehow

Reply:
It seems the issue was the number of names: giving a second name makes this work again.